python/eos/analysis.py

```diff
@@ -217,11 +217,15 @@ def optimize(self, start_point=None, **kwargs):
         """
```

Comment:
I was wondering if we should add a ``rng`` keyword argument here, such that for a random starting point we still have a deterministic behaviour.

Reply:
Definitely!